include/deal.II/base/point.h

```diff
@@ -338,7 +338,9 @@ inline Point<dim, Number>::Point(const Number x, const Number y)
 
 
 template <int dim, typename Number>
-inline Point<dim, Number>::Point(const Number x, const Number y, const Number z)
+inline constexpr Point<dim, Number>::Point(const Number x,
+                                           const Number y,
+                                           const Number z)
```

Comment:
Don't we have problems with `Assert` and `constexpr` with intel, i.e., won't this just cause the same problems as we had in #7134?